.github/workflows/deploy-to-aws.yml

```diff
@@ -16,9 +16,6 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
-        with:
-          ref: ${{ env.AWS_ENVIRONMENT }}
-          # Enforces only AWS env name matching branches are checkout and deployed.
```

Comment:
This was a pain.. I was making experimental changes in the `pom.xml` file for the new situation, but they were not applied, because I changed the checkout here to force it to `dev` HEAD instead. 
The enforcing of the right environment is also already guaranteed by line 5 and line 10 of this file.